addons/hr/migrations/12.0.1.1/pre-migration.py

```diff
@@ -3,11 +3,16 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 from openupgradelib import openupgrade
 
+_field_renames = [
+    ('hr.employee', 'hr_employee', 'vehicle_distance', 'km_home_work'),
+]
+
 xmlid_renames = [
     ('hr.employee_root', 'hr.employee_admin'),
 ]
 
 
-@openupgrade.migrate(use_env=False)
-def migrate(cr, version):
-    openupgrade.rename_xmlids(cr, xmlid_renames)
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.rename_fields(env, _field_renames)
```

Comment:
You can't do this if you don't have installed `hr_contract` module, so before calling this method, you have to check if the column exists.